src/common/src/config.rs

```diff
@@ -825,6 +830,17 @@ impl AsyncStackTraceOption {
     }
 }
 
+#[derive(Debug, Default, Clone, Copy, ValueEnum, Serialize, Deserialize, EnumAsInner)]
+pub enum StrictConsistencyOption {
+    /// Enabled, panic on inconsistency.
+    #[default]
+    On,
+    /// Disabled, warn on inconsistency.
+    Off,
+    /// Disabled, ignore inconsistency.
+    OffSilent,
```

Comment:
```suggestion
    Ignore,
```

Reply:
By the way, I think we may remove this and only offer `On` and `Off` now. For each use cases that I can think of,

- UDF: It should be *almost* deterministic
- TTL: Ideally, the dataset should have good time locality, so the new data should rarely touch the evicted data.
- Historical software bugs: Should be very few

So I don't think it's necessary to provide an option of ignore slightly.

Reply:
If we setup a longevity test with insane mode enabled, there will be many many such warning logs. I guess for that case we should set to `Ignore`.

Reply:
Thinking... Perhaps `LogSuppressor` would help. Besides, we may also exclude this particular error by setting `RUST_LOG` module scope for longevity test.

Reply:
Makes sense👍

Reply:
> If we setup a longevity test with insane mode enabled, there will be many many such warning logs. I guess for that case we should set to `Ignore`.

What about setting `RUST_LOG` for this case?